apps/datahub/src/assets/img/geo2france.svg

```diff
@@ -0,0 +1,73 @@
+            <path d="M590,414.9L570.7,414.9L570.7,354.6C579,351.6 591.8,348.8 603.4,348.8C621.1,348.8 629.9,356.2 629.9,379.3L629.9,414.8L610.6,414.8L610.6,380.8C610.6,369.1 608.7,366 599.3,366C596.2,366 593.3,366.4 590.1,367.1L590.1,414.9L590,414.9Z" style="fill:rgb(15,68,150);fill-rule:nonzero;"/>
+            <path d="M695.4,413.4C690.2,415.5 681.6,416.7 675.2,416.7C656,416.7 643.2,408.8 643.2,382.4C643.2,356.3 656.1,348.9 675.2,348.9C681.9,348.9 690.4,349.9 695.4,351.7L691.4,368C687.8,366.6 682.1,365.4 676.8,365.4C667.4,365.4 662.6,367 662.6,382.9C662.6,398 667.4,400.2 676.8,400.2C682.3,400.2 687.7,398.8 691.4,397.3L695.4,413.4Z" style="fill:rgb(15,68,150);fill-rule:nonzero;"/>
+            <path d="M762.1,388.4L722.3,388.4C722.6,398.7 727.2,400.6 736.5,400.6C741.1,400.6 749.6,399.7 754.3,398.5L757.2,413.2C750.8,415.3 741.2,416.7 734,416.7C715.9,416.7 702.5,408.8 702.5,382.4C702.5,356.5 714.3,348.9 732.8,348.9C754,348.8 763.9,355.3 762.1,388.4ZM722.4,374.2L743.8,374.2C743.5,365.3 741.2,363.4 733.6,363.4C725.5,363.4 723.5,366 722.4,374.2Z" style="fill:rgb(15,68,150);fill-rule:nonzero;"/>
+        </g>
+    </g>
+    <g id="Calque_11" transform="matrix(1,0,0,1,-51.8858,-42.7)">
+        <rect x="504.8" y="113.2" width="26.3" height="26.3" style="fill:rgb(237,28,36);"/>
+        <rect x="426" y="86.4" width="26.3" height="26.3" style="fill:rgb(15,68,150);"/>
+        <rect x="478.5" y="86.9" width="26.3" height="26.3" style="fill:rgb(15,68,150);"/>
+        <rect x="478.5" y="139.5" width="26.3" height="26.3" style="fill:rgb(141,198,63);"/>
+        <rect x="399.7" y="60.1" width="26.3" height="26.3" style="fill:rgb(237,28,36);"/>
+    </g>
+    <defs>
+        <radialGradient id="_Radial1" cx="0" cy="0" r="1" gradientUnits="userSpaceOnUse" gradientTransform="matrix(72.8075,0,0,72.8075,401.11,128.943)"><stop offset="0" style="stop-color:rgb(230,231,232);stop-opacity:1"/><stop offset="0.3" style="stop-color:rgb(207,208,210);stop-opacity:1"/><stop offset="0.75" style="stop-color:rgb(167,169,172);stop-opacity:1"/><stop offset="1" style="stop-color:rgb(167,169,172);stop-opacity:1"/></radialGradient>
+        <linearGradient id="_Linear2" x1="0" y1="0" x2="1" y2="0" gradientUnits="userSpaceOnUse" gradientTransform="matrix(25.3501,-20.5671,20.5671,25.3501,400.223,87.2502)"><stop offset="0" style="stop-color:rgb(230,231,232);stop-opacity:0.37"/><stop offset="0.11" style="stop-color:rgb(207,208,209);stop-opacity:0.37"/><stop offset="0.34" style="stop-color:rgb(149,148,149);stop-opacity:0.37"/><stop offset="0.67" style="stop-color:rgb(57,54,55);stop-opacity:0.37"/><stop offset="0.75" style="stop-color:rgb(35,31,32);stop-opacity:0.37"/><stop offset="1" style="stop-color:rgb(35,31,32);stop-opacity:0.37"/></linearGradient>
+    </defs>
+</svg>
```

Comment:
This cannot be the default placeholder, it has to be something more generic (e.g. the current one)

Reply:
Yep, I agree